test/integration/multifile.spec.ts

```diff
@@ -70,21 +69,19 @@ describe("Multiple-file project instrumentation", () => {
                     dirName,
                     /.+\.sol.instrumented.expected$/
                 );
-
                 expectedInstrumented = new Map(
                     expectedInstrumentedFiles.map((fileName) => [
                         fileName,
                         fse.readFileSync(fileName, "utf-8")
                     ])
                 );
-
                 expectedFlat = fse.readFileSync(`${dirName}/flat.sol.expected`, {
                     encoding: "utf-8"
                 });
-
-                expectedInstrMetadata = fse.readJsonSync(
-                    `${dirName}/instrumentationMetadata.json.expected`,
-                    { encoding: "utf-8" }
+                expectedInstrMetadata = JSON.parse(
```

Comment:
`fse.readJsonSync()` is really better here. So I suggest to restore it.

Reply:
Fixed in dcf955c.